src/Symfony/Component/AssetMapper/AssetMapperRepository.php

```diff
@@ -161,7 +162,9 @@ private function getDirectories(): array
                 continue;
             }
 
-            throw new \InvalidArgumentException(sprintf('The asset mapper directory "%s" does not exist.', $path));
+            if ($this->strictMode) {
```

Comment:
we need this check on line 152 also, don't we?

Reply:
Line 166 is what threw when the application was set to `prod` and the `assets/` directory was missing. I erred on the side of caution and only put the check around that, rather than around both. I can put this check on line 152 as well though!

Reply:
I've added the check to line 152 as well!